vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SlideManagerTest.java

```diff
@@ -0,0 +1,143 @@
+        Assert.assertEquals(actualSlideIdSequence, expectedSlideIdSequence);
+        Assert.assertEquals(slideSequencePresent.size(), sequencesList.size());
+
+    }
+
+    @Test
+    public void test_getSlideSequence_slideNotExistsInSequence() {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        List<Sequence> actualSequenceSlideListNotPresent = slideManagerToTest.getSlideSequences(slideIdNotInSequence,
+                moduleId);
+        Assert.assertTrue(actualSequenceSlideListNotPresent.isEmpty());
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdPresent() throws SlideDoesNotExistException {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        Slide slide = new Slide();
+        slide.setId(slideId);
+        Optional<Slide> slideObj = Optional.of(slide);
+        Mockito.when(slideRepo.findById(slide.getId())).thenReturn(slideObj);
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.verify(slideRepo).delete((Slide) slide);
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdNotPresent() throws SlideDoesNotExistException {
+
+        Mockito.when(slideRepo.findById(Mockito.anyString())).thenReturn(Optional.empty());
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.doThrow(SlideDoesNotExistException.class).when(slideRepo).delete(null);
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdPresentManySequences() throws SlideDoesNotExistException {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+
+        sequenceObjOther.setId(sequenceIdOther);
+        sequenceObjOther.setSlides(slidesList);
+
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        sequencesList.add(sequenceObjOther);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        Slide slide = new Slide();
+        slide.setId(slideId);
+        Optional<Slide> slideObj = Optional.of(slide);
+        Mockito.when(slideRepo.findById(slide.getId())).thenReturn(slideObj);
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.verify(slideRepo).delete((Slide) slide);

```

Comment:
a) you also want to make sure the slide has been removed from the sequences.
b) you probably want another slide object in at least one of the sequences to make sure not all slides are being deleted.